app/controllers/support_interface/application_forms/applicant_details_controller.rb

```diff
@@ -0,0 +1,35 @@
+module SupportInterface
+  module ApplicationForms
+    class ApplicantDetailsController < SupportInterfaceController
+      def edit
+        @details = EditApplicantDetailsForm.new(
+          ApplicationForm.find(params[:application_form_id]),
+        )
+      end
+
+      def update
+        @details = EditApplicantDetailsForm.new(
+          ApplicationForm.find(params[:application_form_id]),
+        )
+
+        @details.phone_number = edit_application_params[:phone_number]
```

Comment:
This might work:

```suggestion
        @details.assign_attributes(edit_application_params)
```

It removes the duplication with `edit_application_params`, so that if we add new attributes they only need to be added once.